CHANGELOG.md

```diff
@@ -1,5 +1,48 @@
 # Change Log
 
+## v0.8.4 2020-12-08
+
+This release includes a number of improvements and bug fixes
+
+[full changelog](https://github.com/executablebooks/jupyter-book/compare/v0.8.3...6c30f554d86fe7d1a0e4ad05012a5de4133117d0)
+
+### New
+
+* 👌 IMPROVE: Option to exclude every file not in the toc [#1123](https://github.com/executablebooks/jupyter-book/pull/1123)
+* 👌 IMPROVE: enable custom builder passthrough [#1094](https://github.com/executablebooks/jupyter-book/pull/1094)
+* 👌 IMPROVE: adding dirhtml builder [#1092](https://github.com/executablebooks/jupyter-book/pull/1092)
+* ✨ ENH: Enable local_extensions to be specified in _config.yml [#1102](https://github.com/executablebooks/jupyter-book/pull/1102)
+* ✨ NEW: add `--individualpages` option for pdflatex builder [#944](https://github.com/executablebooks/jupyter-book/pull/944)
+
+### Upgrades
+
+* ⬆ UPGRADE: sphinx-book-theme v0.0.39 [#1086](https://github.com/executablebooks/jupyter-book/pull/1086)
+
+### Bugs fixed
+
+* 🐛 FIX: Check for file extensions when generating toc [#1108](https://github.com/executablebooks/jupyter-book/pull/1108)
+* 🐛 FIX: Export Notebook as HTML with no page-breaks [#903](https://github.com/executablebooks/jupyter-book/pull/903)
+* 🐛 FIX: Restore linkcheck to builder opts [#1051](https://github.com/executablebooks/jupyter-book/pull/1051)
+
+### Deprecated
+
+* 🗑 DEPRECATE: removing expand_sections for toc [#1073](https://github.com/executablebooks/jupyter-book/pull/1073)
+
+### Docs
```

Comment:
I think you can basically remove all of this section; it's not giving the people that read this any useful information (if they want to see all the commits they can use GitHub). 

Instead, in the improvements section add links to the new documentation, where people can read more about the new features. 

The only item that you may want to mention specifically is the new book gallery (with a link) to inform people about it.